lib/model/database.dart

```diff
@@ -53,8 +53,9 @@ class AppDatabase extends _$AppDatabase {
 
   // When updating the schema:
   //  * Make the change in the table classes, and bump schemaVersion.
-  //  * Export the new schema and generate test migrations:
+  //  * Export the new schema and generate test migrations with drift:
   //    $ tools/check --fix drift
+  //    and generate database code with build_runner.
```

Comment:
Is there a specific command we can put here, or perhaps refer the reader to some documentation, even if it's just `dart run build_runner --help`? (The `dart run` seems helpful to a contributor arriving here for the first time.)